homeassistant/components/withings/__init__.py

```diff
@@ -202,6 +214,9 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload Withings config entry."""
     webhook_unregister(hass, entry.data[CONF_WEBHOOK_ID])
 
+    for callback in hass.data[DOMAIN][entry.entry_id][CALLBACKS]:
+        callback()
```

Comment:
So, instead of doing all of this, you can also just add it to the list of functions that need to be called on entry unload.

Search the codebase for `entry.async_on_unload(` and you will find tons of examples.

Reply:
If only I knew it was this simple